packages/@simple-dom/serializer/src/index.ts

```diff
@@ -15,18 +15,41 @@ const ESC: {
   '>': '&gt;',
 };
 
+const VALID_UPPER_TAGS: {
+  [upper: string]: string;
+} = {
+  DIV: 'div',
+  SPAN: 'span',
+};
+
+export enum WHITESPACE_TRIM_VALS {
+  LEFT = 'trimLeft',
+  RIGHT = 'trimRight',
+  ALL = 'trim',
+}
+
 function matcher(char: string) {
   return ESC[char];
 }
 
 function toLowerCase(name: string) {
-  return name === 'DIV' ? 'div' : name === 'SPAN' ? 'span' : name.toLowerCase();
+  return VALID_UPPER_TAGS[name] || name.toLowerCase();
+}
+
+export interface HTMLSerializerOpts {
```

Comment:
I'd personally prefer to avoid the abbreviation here (and below), what do you think about this?

```suggestion
export interface HTMLSerializerOption {
```